modules/installation-configuration-parameters.adoc

```diff
@@ -1591,8 +1603,76 @@ Additional VMware vSphere configuration parameters are described in the followin
 |====
 |Parameter|Description|Values
 
+|`platform.vsphere`
+| Parameter covers two purposes in the context of a vSphere configuration: (1) Describes your account on the cloud platform that hosts your cluster, and you use the parameter to customize the platform; (2) The parameter is optional when providing additional configuration settings for the machine pool parameters associated with compute and control plane machines.
+a|String
+[NOTE]
+====
+You can only specify one vCenter server for your {product-title} cluster.
+====
+
+|`platform.vsphere.vCenters`
```

Comment:
platform.vsphere.vCenters is not a valid field
```
sh-4.4$  ./openshift-install explain installconfig.platform.vCenters
FATAL Error executing openshift-install: failed to load schema for the field platform.vCenters: invalid field vCenters, no such property found
```
see 
```
sh-4.4$  ./openshift-install explain installconfig.platform.vsphere.vcenters
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <[]object>
  VCenters holds the connection details for services to communicate with vCenter. Currently only a single vCenter is supported.

FIELDS:
    datacenters <[]string> -required-
      Datacenter in which VMs are located.

    password <string> -required-
      Password is the password for the user to use to connect to the vCenter.

    port <integer>
      Default: 443
      Format: int32
      port is the TCP port that will be used to communicate to the vCenter endpoint. This is typically unchanged from the default of HTTPS TCP/443.

    server <string> -required-
      server is the fully-qualified domain name or the IP address of the vCenter server.

    user <string> -required-
      Username is the username that will be used to connect to vCenter
```

Reply:
Updated